server/tests/backends/fixtures/convert/from_datetime_to_text_pypika.json

```diff
@@ -0,0 +1,172 @@
+{
+  "exclude": [
+    "mongo",
+    "mysql",
+    "pandas",
+    "postgres",
+    "snowflake",
+    "athena_pypika",
+    "bigquery_pypika"
```

Comment:
Doesn't this work with Athena and BQ ?

Reply:
different outcomes: 
in Athena & bigquery the` hh:mm:ss are displayed unlike in others.

Reply:
Aaah right, that's because `brewing_date` is a date, not a datetime... I wonder if we should distinguish between the two types later on, or maybe systematically convert everything to the datetime type. Anyway, out of the scope of this PR. Thanks for the new fixture!